SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -857,6 +865,44 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+void FSpatialGDKEditorToolbarModule::UpdateMobileClient() const
+{
+	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
+	const UIOSRuntimeSettings* IOSRuntimeSettings = GetDefault<UIOSRuntimeSettings>();
+	const FString ProjectName = FApp::GetProjectName();
+
+	const FString CommandLineArgs = FString::Printf(TEXT( "../../../%s/%s.uproject %s -game -log -workerType UnrealClient"), *ProjectName, *ProjectName, *(SpatialGDKSettings->ExposedRuntimeIP));
+	const FString Filename = FPaths::Combine(*FPaths::ProjectLogDir(), TEXT("ue4commandline.txt"));
+	if(!FFileHelper::SaveStringToFile(CommandLineArgs, *Filename, FFileHelper::EEncodingOptions::ForceUTF8WithoutBOM))
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Failed to write command line args to file: %s"), *Filename);
+		return;
+	}
+
+	FString DeploymentServerExe = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::EngineDir(), TEXT("Binaries/DotNET/IOS/deploymentserver.exe")));
+	FString DeploymentServerArguments = FString::Printf(TEXT("copyfile -bundle \"%s\" -file \"%s\" -file \"/Documents/ue4commandline.txt\""), *(IOSRuntimeSettings->BundleIdentifier), *Filename);
+	FString DeploymentServerOutput;
+	int32 ExitCode;
+
+#if PLATFORM_WINDOWS
+	FSpatialGDKServicesModule::ExecuteAndReadOutput(DeploymentServerExe, DeploymentServerArguments, FPaths::EngineDir(), DeploymentServerOutput, ExitCode);
```

Comment:
Have you tried using `FPlatformProcess::ExecProcess`?

Reply:
I have, main reason I'm not using it is because `FSpatialGDKServicesModule::ExecuteAndReadOutput` already does everything to print the output as well, which is important in case it fails